int16 FLSpeed
int16 FRSpeed
int16 BLSpeed
int16 BRSpeed
int16 FLAddEN 
int16 FRAddEN 
int16 BLAddEN 
int16 BRAddEN
int16 Voltage
int16 State    


